src/api/events.ts

```diff
@@ -0,0 +1,17 @@
+import type * as vscode from 'vscode'
+import { event } from './api'
+
+interface EthcodeInterface {
+  network: vscode.EventEmitter<string>
+  account: vscode.EventEmitter<string>
+}
+
+export function ethcode (): EthcodeInterface {
```

Comment:
Let's rename the Events endpoint. As per our previous discussion I would prefer `events`.

Reply:
instead of changing ethcode name everywhere i just changes it over here which works perfectly fine if not i will change it everywhere 
![image](https://user-images.githubusercontent.com/72294477/221408951-8da49d16-e499-4049-90b6-bd7a747672c3.png)